geps/gep-713/index.md

```diff
@@ -1,1157 +1,1094 @@
-
-Probably the biggest impediment to this GEP moving forward is the discoverability
-problem; that is, it’s critical that an object owner be able to know what policy
-is affecting their object, and ideally its contents.
-
-To understand this a bit better, let’s consider this parable, with thanks to Flynn:
-
-### The Parable
-
-It's a sunny Wednesday afternoon, and the lead microservices developer for
-Evil Genius Cupcakes is windsurfing. Work has been eating Ana alive for the
-past two and a half weeks, but after successfully deploying version 3.6.0 of
-the `baker` service this morning, she's escaped early to try to unwind a bit.
-
-Her shoulders are just starting to unknot when her phone pings with a text
-from Chihiro, down in the NOC. Waterproof phones are a blessing, but also a
-curse.
-
-**Chihiro**: _Hey Ana. Things are still running, more or less, but latencies
-on everything in the `baker` namespace are crazy high after your last rollout,
-and `baker` itself has a weirdly high load. Sorry to interrupt you on the lake
-but can you take a look? Thanks!!_
-
-Ana stares at the phone for a long moment, heart sinking, then sighs and
-turns back to shore.
-
-What she finds when dries off and grabs her laptop is strange. `baker` does
-seem to be taking much more load than its clients are sending, and its clients
-report much higher latencies than they’d expect. She doublechecks the
-Deployment, the Service, and all the HTTPRoutes around `baker`; everything
-looks good. `baker`’s logs show her mostly failed requests... with a lot of
-duplicates? Ana checks her HTTPRoute again, though she's pretty sure you
-can't configure retries there, and finds nothing. But it definitely looks like
-clients are retrying when they shouldn’t be.
+As a pattern, targeting virtual types has prior art in Kubernetes with the Role Based Access Control (RBAC), where Roles and ClusterRoles can used to specify permissions regarding any kind of resource including non-Kubernetes resources.
 
-She pings Chihiro.
+### Conflicting specs and Inheritance
 
-**Ana**: _Hey Chihiro. Something weird is up, looks like requests to `baker`
-are failing but getting retried??_
+Declaring additional specifications to objects from the outside can yield conflicts that need to be addressed in the implementation of metaresources. Multiple instances of a metaresource kind may affect an object (directly or indirectly), thus posing a possible conflict to resolve regarding which intent among the multiple metaresource specs a controller shall honor.
 
-A minute later they answer.
+In some cases, the most recent between two conflicting specs may be desired to win, whereas in other cases it might be the oldest; often the winning spec is determined by the hierarchical level (implicit or explicit) of the context which the metaresource applies, and sometimes other criteria must be adopted to resolve conflicts between metaresources ultimately affecting a same target or section of a target.
```

Comment:
We should declare in this GEP which is the default behavior.  My vote for the default is for the more specific to override the less specific, e.g. for a policy applied to a listener to override a conflicting policy applied to its parent Gateway.

Reply:
Done